x/upgrade/sonar-project.properties

```diff
@@ -0,0 +1,14 @@
+sonar.projectKey=cosmos-sdk-x-upgrade
```

Comment:
can you create the project on sonarcloud @julienrbrt ?

Reply:
Done! https://sonarcloud.io/project/overview?id=cosmos-sdk-x-upgrade